docs/guides/bake/compose-xbake.md

```diff
@@ -0,0 +1,125 @@
+---
+title: "Extension field with Compose"
```

Comment:
Let's just call it "Building Compose files" and add an extra intro block explaining that you can just build compose files directly and how each service definition inside the compose files maps to a target (could have a simple example for this as well).

Reply:
There is already a basic example in "File definition" for compose files. Also in "Target" specification we already have a note saying "In the case of compose files, each service corresponds to a target.".

This "Extension field with Compose" is very specific to the case where some fields are not supported in the compose spec.

Do you think we should move the [Compose file](https://deploy-preview-14644--docsdocker.netlify.app/build/bake/file-definition/#compose-file) section to this page so we don't have many things around about compose but on the same page?

Reply:
I don't think this provides enough context for the new reader. They already need to know that they are looking for `x-bake` to find this useful. You can leave the example in the file definition but there is much more to explain in here.

Reply:
Added some context for building compose files. In a follow-up we could add new use cases to show how to use compose build fields from the specification and how they are translated to target with bake (secrets, cache).